htdocs/adherents/list.php

```diff
@@ -41,6 +41,10 @@
 
 
 // Get parameters
+$catid 		= GETPOST("catid", 'int');
```

Comment:
Most pages follow the example of modulbuilder that set for action+navigation parameters